src/app/pages/home/home.js

```diff
@@ -52,16 +52,14 @@ export default class Home extends BaseView {
             return;
         }
 
-        let secondaryNavHeight = appView.header.secondaryNavHeight;
+        let headerHeight = appView.header.height;
 
-        if (window.pageYOffset > secondaryNavHeight && !appView.header.isPinned()) {
-            let height = appView.header.height;
-
-            appView.header.reset().collapse().pin();
-            this.el.style.paddingTop = `${height / 10}rem`;
-        } else if (window.pageYOffset <= secondaryNavHeight && !appView.header.isTransparent()) {
-            appView.header.reset().transparent();
-            this.el.style.paddingTop = '0';
+        if (window.pageYOffset >= headerHeight + 100) {
+            appView.header.pin().visible();
+        } else if (window.pageYOffset < headerHeight + 100 && window.pageYOffset >= headerHeight + 20) {
```

Comment:
How can we make sure these kind of things get to everyone?

On 02/18/2016 11:44 AM, Derek Kent wrote:
>
> In src/app/pages/home/home.js 
> <https://github.com/openstax/os-webview/pull/31#discussion_r53351443>:
>
> >
> > -        if (window.pageYOffset > secondaryNavHeight && !appView.header.isPinned()) {
> > -            let height = appView.header.height;
> > -
> > -            appView.header.reset().collapse().pin();
> > -            this.el.style.paddingTop = `${height / 10}rem`;
> > -        } else if (window.pageYOffset <= secondaryNavHeight && !appView.header.isTransparent()) {
> > -            appView.header.reset().transparent();
> > -            this.el.style.paddingTop = '0';
> > +        if (window.pageYOffset >= headerHeight + 100) {
> > +            appView.header.pin().visible();
> > +        } else if (window.pageYOffset < headerHeight + 100 && window.pageYOffset >= headerHeight + 20) {
>
> @Fredasaurus <https://github.com/Fredasaurus> indicated to me 
> yesterday that his intent was for the header to scroll as it currently 
> does on https://os-webview-dev.openstax.org (so that it transitions 
> smoothly as soon as you scroll 1px beyond it). I don't think the |+ 
> 100| and |+ 20| are thus applicable/desirable.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/openstax/os-webview/pull/31/files#r53351443>.
>
> !DSPAM:1106,56c60312174191487873057!